app/controllers/api/apps_controller.rb

```diff
@@ -145,6 +145,7 @@ def generate_profile_response(app)
       max_tps: app.max_tps,
       cluster_name: app.cluster_name,
       consul_host: app.consul_host,
+      consul_hosts: infrastructure.infrastructure_components.where(component_type: 'consul').pluck(:ipaddress).map { |ip| "#{ip}:8500" },
```

Comment:
consul port is hardcoded? can we remove it or use function instead?

Reply:
done.